packages/odyssey-icons/.svgo.yml

```diff
@@ -58,8 +58,7 @@ plugins:
   - removeAttrs: false
   - removeAttributesBySelector: false
   - removeElementsByAttr: false
-  - addClassesToSVGElement:
-      classNames: ["ods-icon"]
```

Comment:
I don't think we should remove this yet. The stand alone icons provided by this package are available to be used in other contexts. Without this class (and the accompanying style) they won't display correctly and will need to be edited individually by consuming teams.

```
.ods-icon {
  position: relative;
  inset-block-start: -0.05em;
  width: 1em;
  height: 1em;
  vertical-align: middle;
}
```
specifically the class is responsible for the correct width and height.